src/Autofac.Extras.MvvmCross/Autofac.Extras.MvvmCross.csproj

```diff
@@ -0,0 +1,47 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <Description>Autofac extension supporting MvvmCross dependency injection.</Description>
+    <VersionPrefix>5.0.0</VersionPrefix>
+    <TargetFrameworks>netstandard2.0</TargetFrameworks>
+    <NoWarn>$(NoWarn);CS1591</NoWarn>
+    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+    <GenerateDocumentationFile>true</GenerateDocumentationFile>
+    <AssemblyName>Autofac.Extras.MvvmCross</AssemblyName>
+    <PackageId>Autofac.Extras.MvvmCross</PackageId>
+    <PackageTags>autofac;di;ioc;dependencyinjection</PackageTags>
+    <PackageReleaseNotes>Release notes are at https://github.com/autofac/Autofac.Extras.MvvmCross/releases</PackageReleaseNotes>
+    <PackageIconUrl>https://cloud.githubusercontent.com/assets/1156571/13684110/16b8f152-e6bf-11e5-84ae-22c66c6d351a.png</PackageIconUrl>
+    <PackageProjectUrl>http://autofac.org</PackageProjectUrl>
+    <PackageLicenseUrl>http://www.opensource.org/licenses/mit-license.php</PackageLicenseUrl>
+    <RepositoryType>git</RepositoryType>
+    <RepositoryUrl>https://github.com/autofac/Autofac.Extras.MvvmCross</RepositoryUrl>
+    <PackageTargetFallback Condition=" '$(TargetFramework)' == 'netstandard1.1' ">$(PackageTargetFallback);portable-net45+win8+wp8+wpa81</PackageTargetFallback>
+    <GenerateAssemblyTitleAttribute>false</GenerateAssemblyTitleAttribute>
+    <GenerateAssemblyCompanyAttribute>false</GenerateAssemblyCompanyAttribute>
+    <GenerateAssemblyProductAttribute>false</GenerateAssemblyProductAttribute>
+    <GenerateNeutralResourcesLanguageAttribute>false</GenerateNeutralResourcesLanguageAttribute>
+    <GenerateAssemblyVersionAttribute>false</GenerateAssemblyVersionAttribute>
+    <GenerateAssemblyFileVersionAttribute>false</GenerateAssemblyFileVersionAttribute>
+    <GenerateAssemblyInformationalVersionAttribute>false</GenerateAssemblyInformationalVersionAttribute>
+    <GenerateAssemblyConfigurationAttribute>false</GenerateAssemblyConfigurationAttribute>
+    <GenerateAssemblyCopyrightAttribute>false</GenerateAssemblyCopyrightAttribute>
+    <GenerateAssemblyDescriptionAttribute>false</GenerateAssemblyDescriptionAttribute>
+  </PropertyGroup>
+
+  <ItemGroup Condition=" '$(TargetFramework)' == 'netstandard2.0' ">
+    <PackageReference Include="Autofac" Version="4.8.1" />
+    <PackageReference Include="MvvmCross" Version="6.2.0-beta.2" />
```

Comment:
Why does this project depend on a prerelease version? Wouldn't it be better to aim for the lowest possible version or at least a stable one?

Reply:
The reason it's depending on MvvmCross 6.2.0-beta.2 is because my project depends on features provided by the upcoming MvvmCross 6.2.0 release. So I figured since I'm updating our clone of `Autofac.Extras.MvvmCross` anyway, might as well share it via a PR so others might benefit from it as well. 

But I understand if this won't be released until MvvmCross 6.2.0 is also released, at which point we can update the package reference here.

Reply:
hey there, thanks a lot for this code. So 6.2 is already released, can I help anyhow? we use this package and would love to have a stable version referenced in our project.

Reply:
It looks like we should be able to pull this in and make a few tweaks (version numbers, etc.) to get it out the door.

(If folks _really_ want to help, [we're still looking for project owners for extensions like this](https://www.paraesthesia.com/archive/2018/04/02/autofac-looking-for-extension-owners/)...)